src/site/playground/src/routes/index.tsx

```diff
@@ -110,130 +107,116 @@ function RouteComponent() {
       } else {
         return setWirespecErrors([
           {
-            value: `Invalid OpenAPI specification; missing 'swagger' or 'openapi' property`,
-            line: 1,
-            position: 1,
-            length: 1,
+            message: "Invalid JSON",
+            startLineNumber: 1,
+            startColumn: 1,
+            endLineNumber: 1,
+            endColumn: 1,
           },
         ]);
       }
-      setWirespecErrors([]);
-    } catch (error) {
-      if (error instanceof Error) {
-        setWirespecErrors([
-          { value: error.message, line: 1, position: 1, length: 1 },
-        ]);
-      }
+    } catch (e) {
+      return setWirespecErrors([
+        {
+          message: "Invalid JSON",
+          startLineNumber: 1,
+          startColumn: 1,
+          endLineNumber: 1,
+          endColumn: 1,
+        },
+      ]);
     }
   }, [code, emitter, specification]);
 
   useEffect(() => {
-    if (!monaco) return;
-    initializeMonaco(monaco);
+    if (monaco) {
+      initializeMonaco(monaco);
+    }
   }, [monaco]);
 
   useEffect(() => {
-    if (!monaco) return;
+    if (monaco && wirespecOutput?.errors) {
+      setMonacoErrors(monaco, wirespecOutput.errors);
+    }
+  }, [monaco, wirespecOutput?.errors]);
 
-    setMonacoErrors(monaco, wirespecErrors);
-  }, [wirespecErrors, monaco]);
+  useEffect(() => {
+    if (wirespecOutput?.result) {
+      const result = wirespecOutput.result
+        .map((it) => {
+          return `${createFileHeaderFor(it.typeName, emitter)}${it.result}`;
+        })
+        .join("\n\n");
+      setWirespecResult(result);
+    }
+  }, [wirespecOutput?.result, emitter]);
```

Comment:
I'm not sure what happened here, but I don't think this should be part of this MR tbh

Reply:
Removed again. Junie went roque :o